packages/block-editor/src/hooks/text-align.js

```diff
@@ -0,0 +1,190 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { addFilter } from '@wordpress/hooks';
+import {
+	getBlockSupport,
+	getBlockType,
+	hasBlockSupport,
+} from '@wordpress/blocks';
+import { alignLeft, alignRight, alignCenter } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import { AlignmentControl, BlockControls } from '../components';
+import { useBlockEditingMode } from '../components/block-editing-mode';
+
+const TEXT_ALIGN_SUPPORT_KEY = 'typography.textAlign';
```

Comment:
In other typography supports, such keys are exported and are part of [TYPOGRAPHY_SUPPORT_KEYS](https://github.com/WordPress/gutenberg/blob/5c84e9fb3192de714da35afc0ed8b665ae83b57e/packages/block-editor/src/hooks/typography.js#L37). I'm a little unsure if we should also export this key.

Since this block support just changes the class name, does it not require export?

Reply:
I believe `TYPOGRAPHY_SUPPORT_KEYS` is used in general checks for whether a block has any typography related supports. Those checks though are likely geared towards whether the typography panel should be displayed.

It might be required if this support is to be available within global styles but is more complicated if the typography panel for block instances doesn't get a matching control.